### PR TITLE
Added the payer account billing details

### PR DESCRIPTION
**Pull request by athiruma**
1. Fetch the cost reports of the org-level billing details.

### CODE REVIEW

File: cloud_governance/common/clouds/aws/cost_explorer/cost_explorer_operations.py [hunk 1]
```diff
@@ -11,8 +11,8 @@ class CostExplorerOperations:
     START_DAY = 1
     END_DAY = 31
 
-    def __init__(self):
-        self.cost_explorer_client = boto3.client('ce')
+    def __init__(self, ce_client = ''):
+        self.cost_explorer_client = boto3.client('ce') if not ce_client else ce_client
```
Comment (ebattat) [inline cloud_governance/common/clouds/aws/cost_explorer/cost_explorer_operations.py:15]: Is the syntax properly here ?

File: cloud_governance/policy/aws/cost_explorer_payer_billings.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+import boto3
+from cloud_governance.common.logger.logger_time_stamp import logger_time_stamp
+
+from cloud_governance.common.logger.init_logger import logger
+from cloud_governance.common.clouds.aws.cost_explorer.cost_explorer_operations import CostExplorerOperations
+from cloud_governance.policy.aws.cost_billing_reports import CostBillingReports
+
+
+class CostExplorerPayerBillings(CostBillingReports):
+
```
Comment (ebattat) [inline cloud_governance/policy/aws/cost_explorer_payer_billings.py:13]: Missing class documentation

Comment (ebattat) [reply cloud_governance/policy/aws/cost_explorer_payer_billings.py:13]: Do we need to add some pytest for this class methods ?

File: cloud_governance/common/clouds/aws/cost_explorer/cost_explorer_operations.py (hunk 1 above)
Comment (athiruma) [reply cloud_governance/common/clouds/aws/cost_explorer/cost_explorer_operations.py:15]: yes

File: cloud_governance/main/environment_variables.py [hunk 3]
```diff
@@ -46,7 +46,7 @@ def __init__(self):
                                                                         'nat_gateway_unused',
                                                                         'zombie_snapshots', 'skipped_resources',
                                                                         'monthly_report']
-        self._environment_variables_dict['cost_policies'] = ['cost_explorer', 'cost_over_usage', 'cost_billing_reports']
+        self._environment_variables_dict['cost_policies'] = ['cost_explorer', 'cost_over_usage', 'cost_billing_reports', 'cost_explorer_payer_billings']
```
Comment (ebattat) [inline cloud_governance/main/environment_variables.py:49]: I dont see env variable for user, costcenter

Comment (athiruma) [reply cloud_governance/main/environment_variables.py:49]: No.
I think mam points to Shai's accounts are in the sheet.
If the cost center is in GSheet, I will mark it as Shai as owner.

Comment (ebattat) [reply cloud_governance/main/environment_variables.py:49]: I think that it will be good to have this option, WDYT ?

Comment (athiruma) [reply cloud_governance/main/environment_variables.py:49]: yes good catch.
I added *COST_CENTER_OWNER* this env, so we can give key-value pairs
cost_center: owner

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**